docs/build-parachains-rococo.md

```diff
@@ -0,0 +1,108 @@
+---
+id: build-parachains-rococo
+title: Building Parachains on Rococo
+sidebar_label: Building Parachains on Rococo
+---
+
+[Rococo](https://github.com/paritytech/cumulus#rococo-crown) is a Polkadot testnet built for testing
+parachains. Rococo utilizes Cumulus and HRMP (Horizontal Relay-routed Message Passing) in order to
+send transfers and messages between parachains and the Relay Chain. Every message is sent to the
+Relay Chain, then from the Relay Chain to the desired parachain. Rococo currently runs parachains
+Tick, Trick and Track.
+
+## Who are building Parachains?
+
+As of now, the list of parachains are as follows:
+
+- Rococo (Tick, Trick, and Track)
+- Mandala
+- Darwinia
+- Plasm
+- Robonomics
+- Turbulence
+
+## Parachain Workshop
+
+If you are interested in running and launching your own parachain, Parity Technologies has created a
+[parachain workshop](https://substrate.dev/cumulus-workshop/#/1-prep/1-compiling). There is also a
+[Rococo Element chat channel](https://matrix.to/#/!WuksvCDImqYSxvNmua:matrix.parity.io?via=matrix.parity.io&via=matrix.org&via=web3.foundation)
+as well as [Cumulus' GitHub repository](https://github.com/paritytech/cumulus#rococo) available.
+
+### Obtaining ROC
+
+There currently isn't a faucet available for ROC and disbursements are only limited to parachain
+builders for the time being. It's recommended at this point to run your own parachains testnet
+locally.
```

Comment:
Add a link to the Rococo chatroom as a place for people to ask for Rococo

Reply:
Ah that's already in there =)
Scroll up to "Parachain workshop"